src/scores/continuous/standard_impl.py

```diff
@@ -418,3 +419,137 @@ def pbias(
 
     _pbias = 100 * error.mean(dim=reduce_dims) / obs.mean(dim=reduce_dims)
     return _pbias
+
+
+def kge(
```

Comment:
Does it make sense to add a `weights` arg for if someone wanted to weight the results by latitude?

Reply:
I considered using the weights argument similar to other scores, but I noticed that 'pearsonr' hasn't been implemented with weights yet. Since KGE requires computing 'pearsonr', I will implement the weighted KGE after the weighted 'pearsonr' update."

Reply:
That's reasonable for now. We can include the weights in a subsequent PR. It would be well worth the effort to support the weights argument since it's one of the main features of `scores`, but I think it's quite appropriate to do it incrementally. I will go ahead and create those issues now, and we can move ahead with this implementation for now. If anyone would rather tackle them as part of this change, just say so and we can do it all together instead.